content/docs/rfcs/66/README.md

```diff
@@ -0,0 +1,32 @@
+---
+slug: 66
+title: 66/WAKU2-METADATA
+name: Waku Metadata Protocol
+status: raw
+editor: Alvaro Revuelta <alrevuelta@status.im>
+contributors:
+---
+
+# Metadata Protocol
+
+Waku specifies a req/resp protocol that provides information about the node's medatadata.
+
+
+## Protocol id
+
+`/vac/waku/metadata/1.0.0`
+
+## Request
+
+```proto
+message WakuMetadataRequest {
+}
+```
+
+## Response
+
+```proto
+message WakuMessageResponse {
```

Comment:
Do we want to include `repeated` supported shard numbers here? As a peer, I would still need this information to know if a node is interesting (i.e. can relay/store messages in the shards I'm subscribed to) or is the idea to reduce the set of possible connections sufficiently here and get this information elsewhere (e.g. discovery, hello protocol), etc.?

Reply:
yep good point

Reply:
fixed d381ee1bbdaf033a4561eda773048117e2be731d